addons/mail/static/tests/mock_server/mock_models/res_partner.js

```diff
@@ -148,7 +148,7 @@ export class ResPartner extends webModels.ResPartner {
         for (const partnerId of partners) {
             const data = {
                 name: users[partnerId]?.name,
-                groups_id: users[partnerId]?.groups_id.includes(allowed_group)
+                group_ids: users[partnerId]?.group_ids.includes(allowed_group)
```

Comment:
`all_group_ids` (same change as python comment)

Reply:
Same question as in the Python comment: should I modify both?
```suggestion
                group_ids: users[partnerId]?.all_group_ids.includes(allowed_group)
```

Reply:
Both (if you modify the left one you have to update the JS file I commented on too, which seems already done)

Reply:
Javascript has no knowledge of implied group (and are not supposed to have it). This pr will use group_ids which are the groups that discuss uses as a kind of flag (your javascript test works like this).